src/Makefile

```diff
@@ -84,7 +84,7 @@ phpvendors:
 composer_install:
 	@echo "current dir: '$${PWD}'"
 	$(INSTALL_DATA) composer.json composer.lock $(DESTDIR)$(MODDIR)
-	(cd $(DESTDIR)$(MODDIR); export COMPOSER_HOME=/dev/null; $(COMPOSER_PHAR) install --no-plugins --no-scripts --no-dev)
+	(cd $(DESTDIR)$(MODDIR); $(COMPOSER_PHAR) install --no-plugins --no-scripts --no-dev)
```

Comment:
I think there was a reason for that export. I think it caused permission problems on some systems / when run with specific user.

Reply:
1. I removed the export to get caching capability of composer.
1. Anyway, you need to run this target as root user. So this should not cause any permission issues.

Reply:
There were problems on some systems/users. And if one runs this target for creating a package, it is ran without root privileges. And there is also nothing that would someone prevent from installing as user into a local folder, there are also no root privileges needed. 

This change will break for some users.

Reply:
- I just tried building packages on this branch (using `dpkg-buildpackage -I`) and it went through just fine.
- From the [composer documents](https://getcomposer.org/doc/03-cli.md#composer-home), the `COMPOSER_HOME` by default directs to current user's home directory. And every user have access to their home directory.
- This change will really speed up the source install process.

I can not think of any good reason to revert the change back.

Reply:
Reverting the change here. Will open a separate PR for caching.